htdocs/index.php

```diff
@@ -82,6 +82,9 @@
 $smarty->setCacheDir($cache_dir);
 $smarty->debugging = $smarty_debug;
 
+# Muting warnings due to PHP 8+
+$smarty->muteUndefinedOrNullWarnings();
+
```

Comment:
Seems really interesting, but:
- the method is not available on smarty 3, making service-desk crashing
- on smarty4, we still have a lot of warnings.